src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -21,45 +40,617 @@ const ActorInit En_Reeba_InitVars = {
     (ActorFunc)EnReeba_Update,
     (ActorFunc)EnReeba_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Reeba/EnReeba_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Reeba/EnReeba_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK5, 0x11, 0x09, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0xFFCFFFFF, 0x08, 0x08 }, { 0xFFCFFFFF, 0x00, 0x00 }, 0x01, 0x05, 0x01 },
+    { 20, 40, 0, { 0, 0, 0 } },
+};
+
+extern AnimationHeader D_060001E4;
+extern SkeletonHeader D_06001EE8;
+
+void EnReeba_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnReeba* this = THIS;
+    f32 temp_f0;
+    s32 surfaceType;
+    s32 pad;
+
+    this->actor.naviEnemyId = 0x47;
+    this->actor.unk_1F = 3;
+    this->actor.gravity = -3.5f;
+    this->actor.posRot2.pos = this->actor.posRot.pos;
+    SkelAnime_Init(globalCtx, &this->skelanime, &D_06001EE8, &D_060001E4, this->limbDrawTable,
+                   this->transitionDrawTable, 18);
+    this->actor.colChkInfo.mass = 0xFE;
```

Comment:
I don't know if we ever decided, but mass feels like something that should be decimal? 
```suggestion
    this->actor.colChkInfo.mass = 254;
```

Reply:
mass 0xFE and 0xFF are special casses that we are keeping in hex for now.
We had a debate about a define name but never settled on something tho